### DISCUSSION TITLE
Training Log 2023-01-13

### DISCUSSION THREAD

**Discussion by zh-zheng** [OpenBMB/CPM-Live#306] [Training Logs 训练日志]
# CPM-Live Training Log (January, 13)

Time: January, 13 2023 19:00

Recorder: @zh-zheng

## Loss
- Begin: 2.09
- End: 2.07
	
## Completed Data
- $\approx$ 2208.26GB

## Average Grad Norm
- 1.18

## Progress
- 92.16%

## Comment

Today, let's translate a poem with our model.

**Comment by zibuyu** [comment]
The bright moonlight in front of my bed is like the frost on the ground. I look up at it, and then lower my head to think about home.

interesting!
